From cf9549485af61658eab3e14e0a89db80742eb547 Mon Sep 17 00:00:00 2001
From: Josh Blum <josh@joshknows.com>
Date: Fri, 21 Apr 2017 11:29:30 -0700
Subject: [PATCH 090/131] soapy - correct constant for frequency correction

This is a typo, some modules use the "CORR" string
in the setFrequency(dir, chan, name, value) API
to perform fine frequency adjustments.
Updated modules will use setFrequencyCorrection(),
this is support for backwards compatible implementations.
---
 lib/soapy/soapy_sink_c.cc   | 4 ++--
 lib/soapy/soapy_source_c.cc | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/lib/soapy/soapy_sink_c.cc b/lib/soapy/soapy_sink_c.cc
index 07cf069..b12b8da 100644
--- a/lib/soapy/soapy_sink_c.cc
+++ b/lib/soapy/soapy_sink_c.cc
@@ -175,7 +175,7 @@ double soapy_sink_c::set_freq_corr( double ppm, size_t chan)
     _device->setFrequencyCorrection(SOAPY_SDR_TX, chan, ppm);
     #else
     std::vector<std::string> components = _device->listFrequencies(SOAPY_SDR_TX, chan);
-    if (std::find(components.begin(), components.end(), "COOR") != components.end())
+    if (std::find(components.begin(), components.end(), "CORR") != components.end())
     {
         _device->setFrequency(SOAPY_SDR_TX, chan, "CORR", ppm);
     }
@@ -189,7 +189,7 @@ double soapy_sink_c::get_freq_corr( size_t chan)
     return _device->getFrequencyCorrection(SOAPY_SDR_TX, chan);
     #else
     std::vector<std::string> components = _device->listFrequencies(SOAPY_SDR_TX, chan);
-    if (std::find(components.begin(), components.end(), "COOR") != components.end())
+    if (std::find(components.begin(), components.end(), "CORR") != components.end())
     {
         return _device->getFrequency(SOAPY_SDR_TX, chan, "CORR");
     }
diff --git a/lib/soapy/soapy_source_c.cc b/lib/soapy/soapy_source_c.cc
index c780966..a645361 100644
--- a/lib/soapy/soapy_source_c.cc
+++ b/lib/soapy/soapy_source_c.cc
@@ -176,7 +176,7 @@ double soapy_source_c::set_freq_corr( double ppm, size_t chan )
     _device->setFrequencyCorrection(SOAPY_SDR_RX, chan, ppm);
     #else
     std::vector<std::string> components = _device->listFrequencies(SOAPY_SDR_RX, chan);
-    if (std::find(components.begin(), components.end(), "COOR") != components.end())
+    if (std::find(components.begin(), components.end(), "CORR") != components.end())
     {
         _device->setFrequency(SOAPY_SDR_RX, chan, "CORR", ppm);
     }
@@ -190,7 +190,7 @@ double soapy_source_c::get_freq_corr( size_t chan )
     return _device->getFrequencyCorrection(SOAPY_SDR_RX, chan);
     #else
     std::vector<std::string> components = _device->listFrequencies(SOAPY_SDR_RX, chan);
-    if (std::find(components.begin(), components.end(), "COOR") != components.end())
+    if (std::find(components.begin(), components.end(), "CORR") != components.end())
     {
         return _device->getFrequency(SOAPY_SDR_RX, chan, "CORR");
     }
-- 
2.20.1

